swagger-confluence-client.yaml

```diff
@@ -47,7 +47,7 @@ paths:
   /content/{pageId}:
     get:
       summary: "Gets a confluence page content by id."
-      operationId: "getPage"
+      operationId: "getPageById"
```

Comment:
We tried to use the Atlassian-API-documentation names as operationId. So it should be "getContentById" 
https://docs.atlassian.com/ConfluenceServer/rest/7.1.0/#api/content-getContentById

Reply:
Okay. I' change it